src/Components/CreateDog/CreateDog.js

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState } from "react";
+import { connect, useDispatch } from "react-redux";
+import "./CreateDog.styles.scss";
+import axios from "axios";
+import { setLoading } from "../../Redux/Loading/LoadingSlice";
+import Loading from "../Loading/Loading";
+import { useTranslation } from "react-i18next";
+import InfiniteScroll from 'react-infinite-scroll-component';
+
+const DogCardInfo = ({ isLoading, setLoading }) => {
+  const { t } = useTranslation();
+  const [dogNames, setDogNames] = useState([]);
+  const [dogInfos, setDogInfos] = useState([]);
+  const dispatch = useDispatch();
+  const setTrue = () => {
```

Comment:
setTrue name should be change to something else

Reply:
I will set it to specify what is set to false, in this reference it will be setLoadingTrue.